reference-bot/src/main/java/za/co/entelect/challenge/Bot.java

```diff
@@ -15,14 +14,21 @@
 
 public class Bot {
     private GameState gameState;
-    private Random rand = new Random(LocalTime.now().toNanoOfDay());
 
+    /**
+     * Constructor
+     * @param gameState the game state
+     **/
     public Bot(GameState gameState){
         this.gameState = gameState;
         gameState.getGameMap();
     }
 
-    public String Run(){
+    /**
+     * Run
+     * @return the result
+     **/
+    public String run(){
         String command = "";
 
         //if enemy has an attack building and i dont have a blocking wall, then block from front
```

Comment:
**I**f **the** enemy has an attack building and **I** **don't** have a blocking wall, then block from **the front**